routes/register.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const User = require('../models/User');
+const router = express.Router();
+
+// create a response for the homepage
```

Comment:
I think this comment is meant for `home.js` - this file is for the registration and has nothing to do with the homepage.